src/scripting/lua_root.cc

```diff
@@ -582,6 +607,209 @@ int LuaDescriptions::new_tribe(lua_State* L) {
 	return 0;
 }
 
+/* RST
+   .. method:: modify_unit(type, name, property ... , value ...)
+
+      This powerful function is meant to be used by add-ons of the ``tribes``
+      and ``world`` categories from their ``postload.lua``.
+
+      TODO(Nordfriese): Document
+*/
+// TODO(Nordfriese): I only added functions I need for my own add-ons.
+// If anyone requests the possibility to modify any other tribe-related properties
+// through add-ons, the changes need to go here.
+int LuaDescriptions::modify_unit(lua_State* L) {
```

Comment:
What a moster function! better refactor into smaller functions.

Reply:
Okay will do that